java/src/jmri/jmrix/loconet/LnDeferProgrammer.java

```diff
@@ -15,6 +15,7 @@
  * that might be provided later. This is done by connecting through a LocoNetSystemConnectionMemo.
  *
  * @author Bob Jacobsen Copyright (C) 2018
+ * @author Andrew Crosland Copyright (C) 2021
```

Comment:
It would be good to remove this comment now when the new `readCV` method is removed from this class.